bids/config.py

```diff
@@ -71,7 +70,7 @@ def from_file(filenames, error_on_missing=True):
     if isinstance(filenames, str):
```

Comment:
```suggestion
    if isinstance(filenames, (Path, str)):
```

Reply:
I think instead we'd want:

```suggestion
    if isinstance(filenames, (str, os.PathLike)):
```

Reply:
Thank you! Totally missed this.

I think we don't need to type-check here at all and can instead use `utils.listify`:

```python
filenames = listify(filenames)
```

Reply:
@kalenkovich do you need to make this change still?

Reply:
I did now! I wasn't sure whether I should wait for someone to ok the switch to `listify`.